template/tasks.py

```diff
@@ -207,6 +207,30 @@ def build_image(ctx, tag=None, pty=True):
     )
 
 
+@invoke.task
+def run_image(ctx, tag=None, options: str = None, command: str = None):
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**thought (non-blocking):** You could split out options into a couple of separate optional parameters.

You could add a `network: str|None = None` and `env_file: str|None = None` that would allow a little more control. Not a blocking issue. ‏

Reply:
Thanks for your suggestion! My thought was that by keeping options as a single parameter, we provide more flexibility in the way arguments are passed. However, I appreciate your idea of splitting it into more distinct params. I'll keep it in mind and we can monitor the situation. If the need arises in the future, we can always update this pattern accordingly. 👍🏻